gradle/dependencies.gradle

```diff
@@ -1881,6 +1881,12 @@ ext.libraries = [
                     force = true
                 }
         ],
+
+        pac4jCouchDb             : dependencies.create("org.pac4j:pac4j-couch:$pac4jVersion") {
```

Comment:
Please include this into the pac4j dependency.

Reply:
Why pull in every pac4j data service when we only need one? I only added the one dependency spec because the rest aren't relevant to this.